src/main/java/org/jabref/logic/journals/JournalAbbreviationRepository.java

```diff
@@ -67,7 +69,30 @@ public boolean isAbbreviatedName(String journalName) {
         String journal = journalName.trim();
 
         return customAbbreviations.stream().anyMatch(abbreviation -> isMatchedAbbreviated(journal, abbreviation))
-                || abbreviationToFull.containsKey(journal);
+                || abbreviationToFull.containsKey(journal) || findDottedAbbrFromDotless(journal).length() > 0;
+    }
+
+    public String findDottedAbbrFromDotless(String journalName) {
+        String foundKey = "";
+
+        // check for a dot-less abbreviation
+        if (!Pattern.compile("\\.").matcher(journalName).find()) {
+            // use dot-less abbr to find full name using regex
+            String[] journalSplit = journalName.split(" ");
+
+            for (int i = 0; i < journalSplit.length; i++) {
+                String word = journalSplit[i] + "[\\.\\s]*";
```

Comment:
See my comment above,

Reply:
I'm not sure what you mean.

Did you mean to point out line 91 where it has `Pattern.compile(joined).matcher(s).find()`?

I don't believe I can move `Pattern.compile(joined)` to a static variable since it depends on `String joined = String.join("", journalSplit);` right above it.

Reply:
Ah then I misread that line.